lib/graylog2_exceptions.rb

```diff
@@ -43,13 +43,12 @@ def call(env)
 
   def _call(env)
     begin
-      # Call the app we are monitoring
       response = @app.call(env)
+    rescue SyntaxError => err
```

Comment:
What about other types of errors? 
```
rescue Exception => err
 ...
```

Reply:
Or alternatively, why do we even try to catch errors that are not standard?

Reply:
we just had syntax error in the code and no graylog alerted on that.. 
thats why .. :)